napari/_qt/qt_viewer.py

```diff
@@ -1070,20 +1070,26 @@ def on_draw(self, event):
         This is triggered from vispy whenever new data is sent to the canvas or
         the camera is moved and is connected in the `QtViewer`.
         """
+        # The canvas corners in full world coordinates (i.e. across all layers).
+        canvas_corners_world = self._canvas_corners_in_world
         for layer in self.viewer.layers:
-            if layer.ndim <= self.viewer.dims.ndim:
```

Comment:
I believe this should always be `True` as the number of dimensions in the viewer should always be greater than equal to the number of dimensions in any one layer, but if someone knows better, please correct me and I'll put this back in.